rpc/documentation/api.md

```diff
@@ -1417,6 +1417,21 @@ transaction was seen.
 - `int64 timestamp`: The Unix time of the earliest time this transaction was
   seen.
 
+  **Nested enum:** `TransactionType`
```

Comment:
doco this under the transaction_type field please, this is how it is done elsewhere